internal/constellation/helm/loader.go

```diff
@@ -232,9 +231,17 @@ func (i *chartLoader) loadCertManagerValues() map[string]any {
 		"tolerations": controlPlaneTolerations,
 		"webhook": map[string]any{
 			"tolerations": controlPlaneTolerations,
+			"podDisruptionBudget": map[string]any{
+				"enabled": true,
+			},
+			"replicaCount": 2,
```

Comment:
Cert-Manager has guidance on PDB: https://cert-manager.io/docs/installation/best-practice/#poddisruptionbudget. Is there a reason why we chose different values? Isn't it wasteful to scale to two replicas, if it would be enough to launch an additional one during drains?

Reply:
I just enable the pdb and therefore use the default from their helmcharts, don't I? Also I increase the replica count according to their own warning as stated in the docs you linked.

Reply:
Thanks, I did not realize that `minAvailable: 1` is the default in values.yaml. Ack on the HA comment, this is fine.